core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/defaults/main.yml

```
@@ -5,7 +5,7 @@ cni_plugin_vars:
     canal: k8s-app=canal
     flannel: app=flannel
 
-epiphany_manifests_dir: /etc/epiphany/manifests
+epiphany_manifests_dir: "{{ inventory_dir }}/manifests"
```

```suggestion
epiphany_k8s_manifests_dir: "{{ inventory_dir }}/k8s_manifests"
```
After changing variable name, we will need to adjust all references.